_posts/convars/2020-04-30-mom_api_log_requests_sensitive.md

```diff
@@ -12,4 +12,6 @@ default_value: 0
 
 If enabled, API requests that are sensitive will be logged to console.
 
-<p>!!!!!!! DANGER! Only set this if you know what you are doing! This could potentially expose an API key! !!!!!!!</p>
\ No newline at end of file
+<div class="note warning">
+  <p>!!!!!!! DANGER! Only set this if you know what you are doing! This could potentially expose an API key! !!!!!!!</p>
```

Comment:
You can clean up this warning by removing the extra exclamations and danger, the new warning box should be good enough of an indicator:

```suggestion
  <p>Only set this if you know what you are doing! This could potentially expose an API key!</p>
```